starter-code/app.js

```diff
@@ -1,75 +1,148 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-var app = express();
-
-var index = require('./routes/index');
-var users = require('./routes/users');
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const expressLayouts = require('express-ejs-layouts');
```

Comment:
You are requiring express-ejs-layouts but you don't have it in your package.json. If someone download your code and run 'npm install' the app is not going to work. Have you done a 'npm install bcrypt --save'?